### PR TITLE
{bio}[intel/2018a] Kraken v1.0, Jellyfish v2.2.10, wget v1.19.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2102.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cb0d4af7ae3798df10e54c5978a865b5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/47711109b06b8dbcf88605589e6a0752 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/66aed4a29abef2d8ffabf60213baf45e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2037.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c8aa0550ba010fd31244934637eacfed for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6b971381f266112d6abc75bc3f2e6601 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2062.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3a9a02e9bb20e146f548ed25ad6fbd1d for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/691920f419c56d77ed5f2464938d5b5e for a full test report.

File: easybuild/easyconfigs/w/wget/wget-1.19.4-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ the most widely-used Internet protocols. It is a non-interactive commandline tool,
+ so it may easily be called from scripts, cron jobs, terminals without X-Windows support, etc."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['93fb96b0f48a20ff5be0d9d9d3c4a986b469cb853131f9d5fe4cc9cecbc8b5b5']
+
+builddependencies = [('binutils', '2.28')]
```
Comment (vanzod) [inline easybuild/easyconfigs/w/wget/wget-1.19.4-GCCcore-6.4.0.eb]: Missing `pkg-config` as build dependency causes configure to fail.

**Comment by boegel** [comment]
@vanzod After including `pkg-config` as a build dep, `wget` now fails to find the system OpenSSL during configure:

```
checking for openssl... no
configure: error: Package requirements (openssl) were not met:

No package 'openssl' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.
```

So we'll need to inject the system location in `$PKG_CONFIG_PATH`? Thoughts @vanzod?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2700.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/720b59df88c482ab9536bdf1763d5a1b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2058.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/69630066f8fbf7df7a634d30da7ecacb for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/b27ec351cd15bc4cf6a2ecb3ab96c38b for a full test report.

**Comment by vanzod** [comment]
@boegel I am puzzled by the paths you passed to `PKG_CONFIG_PATH`. Should't it include the paths to the `openssl` library?

**Comment by boegel** [comment]
@vanzod `$PKG_CONFIG_PATH` is the location(s) `pkg-config` should check for `*.pc` files. It'll find `openssl.pc` in `/usr/lib64/pkgconfig`, which specifies where to find the `OpenSSL` libraries:

```
$ cat /usr/lib64/pkgconfig/openssl.pc
prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib64
includedir=${prefix}/include

Name: OpenSSL
Description: Secure Sockets Layer and cryptography libraries and tools
Version: 1.0.2k
Requires: libssl libcrypto
```

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/0f6889bba47cc5ded3e24234935a8cb5 for a full test report.

**Comment by boegel** [comment]
@vanzod Please re-test this?

**Comment by vanzod** [comment]
That paths do not seem to be the right ones on Debian. I will get back to it once I will be back. Remind me if I don't.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/a794953037c8154f2ce7e838c3d135d6 for a full test report.

**Comment by boegel** [comment]
@vanzod Please elabortate on the `pkg-config` paths?

**Comment by vanzod** [comment]
@boegel For Debian based distros you need to add `/usr/lib/x86_64-linux-gnu/pkgconfig` to `PKG_CONFIG_PATH`, otherwise `configure` cannot find `openssl`.

**Comment by boegel** [comment]
@vanzod fixed, please submit a test report?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/9ed699fbb09861a0b8aac233420dfa73 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**